src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Members.cs

```diff
@@ -559,16 +559,16 @@ void visitFunctionPointerSignature(IMethodSymbol symbol)
                         switch (symbol.CallingConvention)
                         {
                             case SignatureCallingConvention.CDecl:
-                                builder.Add(CreatePart(SymbolDisplayPartKind.ClassName, symbol, "Cdecl"));
+                                builder.Add(CreatePart(SymbolDisplayPartKind.Keyword, symbol, "Cdecl"));
```

Comment:
I think I should pass a `null` symbol for these if the team agreed they should be keywords.

@333fred @CyrusNajmabadi Please let me know how I should proceed.

Reply:
i would honestly just revert the compielr change, and do everything on the IDE side.  i feel like we've spent an inordinate amount of time on something absolutely trivial and this is going in circles.  let's make this simple and easy:

1. no compiler changes.  that definitely simplifies things.
2. make the IDE operate in line with teh compiler today.  also very simple on the IDE side.

bing bang boom.

Reply:
@CyrusNajmabadi Alright. Will do that. I may take few days since my next exam is in few days.

Reply:
@CyrusNajmabadi I'm now always classifying as class. Can you review please? Thanks!